rootston/xwayland.c

```diff
@@ -63,6 +61,85 @@ static void handle_request_configure(struct wl_listener *listener, void *data) {
 		xwayland_surface, event->x, event->y, event->width, event->height);
 }
 
+// XXX Needs deep refactoring to get this better. We need to select the correct
+// seat based on seat pointer focus, but interactive moving and resizing is not
+// yet seat aware. Even then, we can only guess because X11 events don't give us
+// enough wayland info to know for sure.
```

Comment:
Make sure this is captured in a ticket

Reply:
This is a general multi-seat issue which I will make a ticket for probably today.